src/NHibernate.Test/NHSpecificTest/GH1228/Fixture.cs

```diff
@@ -7,126 +7,95 @@ public class Fixture : BugTestCase
 		[Test]
 		public void TestOk()
 		{
-			using (ISession s = OpenSession())
+			using var s = OpenSession();
 			{
```

Comment:
I find it confusing to use a block code after an "instruction line" using.

```suggestion
			using var s = OpenSession();
```